Public/Src/App/Bxl/ServerDeployment.cs

```diff
@@ -232,8 +232,15 @@ private static Possible<Unit> TryKillProcess(string assemblyFullPath)
         /// </summary>
         public static string GetDeploymentCacheHash(string deploymentDir)

```

Comment:
>public [](start = 8, length = 6)

Suggest making this private since it sits behind IsServerDeploymentOutOfSync() now